### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by 2bleO**
In this pr we:
-Create an index.js file and set an array of some simple to do tasks.

-Write function to iterate over the tasks array and populate an HTML list item element for each task.

-On page load render the dynamically created list of tasks in the dedicated placeholder.

-Create a style.css and set rules for the To Do List.

### CODE REVIEW

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @2bleO , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] You did a very impressive work. The project fulfills all the requirements of this milestone. Very functional and organized code. Yet, you missed one important thing which is that you left your readme empty. Although you can work on your readme until the last milestone, it is mandatory to have a readme at least with the Microverse like template structure and some instruction on how to setup and run the project.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+# to-do-list
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> Web page to Showcase an Interactive To-do List as Microverse Project.
+
+![screenshot](src/Snapshot.png)
+
+[Online Version]()
+
```
Comment (carlos-ssh) [inline README.md:11]: You're missing information in your README file. You could use [this template](https://github.com/microverseinc/readme-template).

- Prerequisites
- Setup
- Install
- Usage

Kindly work on these changes and request another code review in your dashboard.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @2bleO, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (2bleO) [reply README.md:11]: Empty headings?? Really????  🤦

Comment (carlos-ssh) [reply README.md:11]: - Kindly in your General requirements of your To Do list: list structure you will find this section:
Make sure that you documented your work [in a professional way.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- Kindly check this point:
 5.- Add descriptive README file to your project - please use this[ template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
 
Any question that you have or if you fell stuck, I can support you on slack.
Greetings!.
@carlos-ssh 👾🥷🏽👨🏽‍💻

**Comment by Davidosky007** [comment]
Hi @2bleO, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## [OPTIONAL]

* As stated by the previous reviewers your `Readme` file needs to be more descriptive because it explains the details of your project in a short way. So when another engineer reads that project, they can understand the project just in 5 minutes without having to look to the source code, what is the project used for
* You can add `setup`, `install`, `usage` information of your project 
 * I strongly suggest you address this 🙏 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**